slm_porosity/trajectory.txt

```diff
@@ -0,0 +1,3 @@
+1.0 0.0 0.0 5.0 0.0 0.0
```

Comment:
I suggest to use optional parameters for axis, i.e. `1 6` instead of this line.

Reply:
As I understand you want to check how many parameters are in trajectory file and decrease manual input routine. 
It is a good idea, but it goes against the idea of automatically generated trajectory file and format style note above

Reply:
1. No, check the number of parameters in each line.
2. What is the generator of the trajectory file?
3. What is the problem with it?